rootfs_linux.go

```diff
@@ -100,7 +103,26 @@ func mount(m *configs.Mount, rootfs, mountLabel string) error {
 			return err
 		}
 		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
-	case "tmpfs", "mqueue", "devpts", "sysfs":
+	case "tmpfs":
+		if stat, err = os.Stat(dest); err == nil {
```

Comment:
This should be `err != nil` right ? because we only want to create the directory if it's not there.